db/schema.rb

```diff
@@ -18,7 +18,6 @@
 
   create_table "accounts", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
     t.string "account_number"
-    t.boolean "is_internal"
```

Comment:
This isn't how you do it, you need to create a [migration](https://guides.rubyonrails.org/active_record_migrations.html).